app/controllers/dashboards_controller.rb

```diff
@@ -1,11 +1,13 @@
-# typed: true
+# typed: false
```

Comment:
`verify_authorized` made me do it.  would be nice (as with the exception handling in `ApplicationController` if we could tell sorbet "no really, `verify_authorized` will work".  i think that's the only thing preventing this file from being `typed: strict`.